pennylane/templates/subroutines/__init__.py

```diff
@@ -25,3 +25,4 @@
 from .permute import Permute
 from .qpe import QuantumPhaseEstimation
 from .qmc import QuantumMonteCarlo
+from .all_singles_doubles import AllSinglesDoubles
```

Comment:
You mean for the docs @ixfoduap? 

Because the user should never see in which folder they sit...so one could just add a seperate section to the docs, without any folder/import changes?

Reply:
Yes, I mean the docs website, so here https://pennylane.readthedocs.io/en/latest/introduction/templates.html

We have embeddings, layers, state preparation, etc. templates. I would like to have a section with quantum chemistry templates